### PR TITLE
add PirateVerse faucet

### PR DESCRIPTION
**Pull request by khaihkd**
This pull request is for PirateVerse subnet. 
We are a new gamefi project and have just launched our own PirateVerse Subnet. 
Project info can be found on https://pirateverse.finance/

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -106,6 +106,22 @@
...
             }
+        },
+        {
+            "ID": "PirateVerse",
+            "NAME": "PirateVerseTestnet",
+            "TOKEN": "PVF",
+            "RPC": "https://rpc-testnet.pirateverse.finance/",
+            "CHAINID": 56766,
+            "EXPLORER": "https://testnet.piratescan.io",
+            "IMAGE": "/pirateverse.png",
```
Comment (rajranjan0608) [inline config.json]: It will be nice to have a URL for the image here, instead of accessing it from the public directory.

**Comment by rajranjan0608** [comment]
LGTM! Will approve and merge it soon. Can you also fund the faucet address before we can merge?

**Comment by khaihkd** [comment]
Hi @rajranjan0608 
I just update the config to use an image URL. 
when I made this pull request, I have already sent 50k PVF token to the faucet address 
https://testnet.piratescan.io/tx/0xe0d2eac934a5fb21536c411f4dea2378a0e388f2209d42b9edae68f16599c2a0

**Comment by khaihkd** [comment]
just sent 450k more
https://testnet.piratescan.io/tx/0x3fe9ebcdf98029eee891c7860a4e5ab2dd68787628ef9a8ad07c093c3584b5cd

**Comment by rajranjan0608** [comment]
Awesome. Thanks for the update!

**Comment by khaihkd** [comment]
Awesome. Thank you so much

**Pull request merged**